content/10.diagnostics.md

```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
+One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally [@doi:10.1126/science.abf4063]. 
+After 1 month, 98% of patients were seropositive for spike IgG. 
+Moreover, spike IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, whereby 90% of subjects were seropositive at 6 months.
+Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.
+
+Memory B cells and memory T cells were also measured.
+SARS-CoV-2 spike-specific memory B cell levels steadily rose over the first 120 days post-symptom onset. 
+RBD-specific memory B cells were detected in COVID-19 patients 90 days post-symptom onset in previous studies [@doi:10.1016/j.cell.2020.11.029; @doi:10.1126/sciimmunol.abf8891], this study confirms these finding and shows that these cells steadily increase for over 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
+Likewise, nucleocapsid-specific memory B cells seem to follow a similar pattern [@doi:10.1126/science.abf4063].
+SARS-CoV-2 memory CD8+ T cells were also detected in 70% of 169 COVID-19 patients after 1 month [@doi:10.1126/science.abf4063], which is consistent with previous research [@doi:10.1016/j.cell.2020.05.015].
+However, SARS-CoV-2 memory CD8+ T cells were slightly decreased (50%) 6 months post-symptom onset. 
+In this same subset of COVID-19 patients, 93% of subjects had detectable levels of SARS-CoV-2 memory CD4+ T cells, of which 42% had > 1% SARS-CoV-2-specific CD4+ T cells.
+At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4+ T cells.
+Indeed, the spike-specific memory CD4+ T cells abundance over time were similar to the overall SARS-CoV-2-specific CD4+ T cells [@doi:10.1126/science.abf4063].
+T cell immunity to SARS-CoV-2 at 6-8 months post-symptom onset has also been confirmed by others [@doi:10.1101/2020.11.01.362319; @doi:10.1101/2020.12.08.416636].
+In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never before exposed to SARS-CoV-2. 
+This indicates that there is potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating coronaviruses that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.
+There is concern that immunity may wane over time as there have been several reported cases of reinfection confirmed via genomic analysis of variants associated with infection in patients [@doi:10.1016/S1473-3099(20)30764-7; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
```

Comment:
```suggestion
There is concern that immunity may wane over time, as there several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
```
Does this work? I don't think we go into the evolution in this paper, so trying to just make a nod to how they were able to tell!

Reply:
There is concern that immunity may wane over time, as several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient 

Delete "there"before several I think

What they mean by the genomic analysis here is that there was slight differences between the SARS-CoV-2 variant that caused the patients first infection, versus the second infection. They aren't necessarily referring to a different lineage such as B.1.1.7 etc. They just mean that they are sure it was a different infection. Does that make sense?